contrib/experimental/great_expectations_experimental/expectations/expect_queried_table_row_count_to_be.py

```diff
@@ -0,0 +1,149 @@
+"""
+This is an example of a Custom QueryExpectation.
+For detailed information on QueryQExpectations, please see:
+    https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_query_expectations
+"""
+
+from typing import Any, Dict, Optional, Union
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.exceptions.exceptions import (
+    InvalidExpectationConfigurationError,
+)
+from great_expectations.execution_engine import ExecutionEngine
+from great_expectations.expectations.expectation import (
+    ExpectationValidationResult,
+    QueryExpectation,
+)
+
+
+class ExpectQueriedTableRowCountToBe(QueryExpectation):
+
```

Comment:
I would remove this empty line before the docstring as it goes against our style guide.